change/@ni-nimble-components-83fac5bb-4bc6-4067-951d-def8c7404d0c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
If you can remember, could you take a look at the Chromatic diffs for systemlink-lib-angular once the Renovate PR uptakes this change? (Note that if there are diffs, approving them on the PR build doesn't seem to reset the baseline, you'll have to approve them on the subsequent build of master.

https://www.chromatic.com/builds?appId=62431f379ac921003aaca05a

And your call whether it's worth informing clients of this change with an announcement or post to the UI WG. If it's just 1-2 pixel diffs then maybe not. But could also be a nice teaching opportunity to educate devs about box-sizing CSS.

Reply:
> If you can remember, could you take a look at the Chromatic diffs for systemlink-lib-angular once the Renovate PR uptakes this change?

Will do.

> And your call whether it's worth informing clients of this change with an announcement or post to the UI WG. If it's just 1-2 pixel diffs then maybe not. But could also be a nice teaching opportunity to educate devs about box-sizing CSS.

Don't think it's super interesting yet. Based on the chromatic diffs pretty much only internals of controls are impacted or if you set a border on our controls manually (but I don't expect users to do that, or for us to support that behaving well).

Would be interesting if we aren't consistent in our apps or Angular / material isn't applying page level `box-sizing: border-box` config. I'll put it my back pocket to take a look and see if it's worth discussing but not planning to discuss it yet.

Reply:
Accepted the diff, only manifested in a one pixel tweak in the details pane: https://www.chromatic.com/build?appId=62431f379ac921003aaca05a&number=8583